.isort.cfg

```diff
@@ -10,3 +10,4 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
+known_third_party = dateutil,lxml,mock,psycopg2,setuptools
```

Comment:
This should be restored to previous state

Reply:
pre-commit all times adds this line :/

Reply:
If you haven't rebased, probably. And make sure you have latest pre-commit installed in your repo (`pre-commit install -f`)

Reply:
Don't blame him :stuck_out_tongue_winking_eye:. pre-commit config is outdated in this repo:

https://github.com/OCA/server-ux/blob/332cc53946135e9da153ba1afd7f99a4fd65fa1d/.pre-commit-config.yaml#L54-L57

Reply:
How can that be?

Reply:
@sbidoul did the mass update, isn't it? Did you forget this repo @sbidoul? :thinking:

Reply:
@yajo I did not forget, but apparently the update failed and I did not notice it. See #155